pkg/interfaces/contracts/vault/IBatchRouter.sol

```diff
@@ -93,6 +93,57 @@ interface IBatchRouter {
         bytes calldata userData
     ) external payable returns (uint256[] memory pathAmountsIn, address[] memory tokensIn, uint256[] memory amountsIn);
 
+    /**
```

Comment:
Could add a header here for "ERC4626 Pools," and say generally that these functions facilitate liquidity operations using standard tokens on ERC4626 pools, which contain only wrapped tokens. So we are abstracting away the underlying wrap/unwrap interactions, allowing them to do in a single transparent operation what they would otherwise need to build into a batch operation.

```
    /***************************************************************************
                                   ERC4626 Pools
    ***************************************************************************/

    // These functions allow interacting with ERC4626 Pools (which are composed of wrapped ERC4626 tokens) using only
    // underlying standard tokens. For instance, with `addLiquidityUnbalancedToERC4626Pool` it is possible to add
    // liquidity to an ERC4626 Pool with [waDAI, waUSDC], using only DAI, only USDC, or an arbitrary amount of both.
    // If the ERC4626 buffers in the Vault have liquidity, these will be used to avoid wrapping/unwrapping through
    // the wrapped token interface, saving gas.
    //
    // For instance, adding only DAI to the pool above (and assuming an aDAI buffer with enough liquidity), would
    // pull in the DAI from the user, swap it for waDAI in the internal Vault buffer, and deposit the waDAI into the
    // ERC4626 pool: 1) without having to do any expensive ERC4626 wrapping operations; and 2) without requiring the
    // user to construct a batch operation containing the buffer swap.
```